src/doc/html.md

```diff
@@ -30,9 +30,20 @@ attribute to `<html>` as in this example:
 <html class="no-js" lang="en">
 ```
 
+
+## Document Title
```

Comment:
Sent from Samsung tablet.


-------- Original message --------
From: Arthur Verschaeve <notifications@github.com> 
Date: 07/09/2015 7:32 PM (GMT+08:00) 
To: h5bp/html5-boilerplate <html5-boilerplate@noreply.github.com> 
Cc: Maribeth Aller <bethaller23@gmail.com> 
Subject: Re: [html5-boilerplate] Doc: Minor Update (#1720) 

In src/doc/html.md:

> @@ -30,9 +30,20 @@ attribute to `<html>` as in this example:
>  <html class="no-js" lang="en">
>  ```
>  
> +
> +## Document Title
This makes me wonder how far our documentation should go and at what point it starts to look like an intro to HTML. This looks like general knowledge which would better fit in a getting started with web development tutorial then in the H5BP docs.

On the other hand, it's important to emphasise things like this. What does the rest think?

—
Reply to this email directly or view it on GitHub.